packages/react-components/package.json

```
@@ -22,7 +22,7 @@
     "start": "yarn storybook",
     "docs": "api-extractor run --config=config/api-extractor.local.json --local",
     "build:local": "tsc -p . --module esnext --emitDeclarationOnly && node ../../scripts/typescript/normalize-import --output dist/react-components/src && yarn docs",
-    "storybook": "start-storybook --port 3000 -s ./public",
+    "storybook": "start-storybook --port 3000 -s ./public --no-manager-cache",
```

```suggestion
    "storybook": "start-storybook --port 3000 -s ./public",
```

I don't think that we should have there: it had sense during local development, but not for everyone.